qmlti/lib.php

```diff
@@ -275,10 +314,30 @@ function init_db($db) {
         $sql = 'CREATE TABLE IF NOT EXISTS ' . TABLE_PREFIX . 'lti_outcome ' .
                '(result_sourcedid VARCHAR(255),' .
                ' score VARCHAR(255),' .
+               ' report_url VARCHAR(255),' .
                ' created DATETIME)';
         $ok = $db->exec($sql) !== FALSE;
       }
 
+      if ($ok) {
+        $sql = 'CREATE TABLE IF NOT EXISTS ' . TABLE_PREFIX .  LTI_Data_Connector::REPORTS_TABLE_NAME . ' ' .
```

Comment:
Right - the lti_outcome table (like the outcome.php page) are both test-systems made to mimic the effect of Moodle systems. By doing so, we can test using the test_harness system implemented instead of using a 3rd party tool consumer.

So, to answer your question, the change does nothing benefiting us (haha) because Moodle won't know to pick up the report_url anyway. Thanks for pointing this out, the added attribute will be removed.